src/Node/State.php

```diff
@@ -42,9 +43,14 @@ public static function CANCELLED(): self
         return new self(self::CANCELLED);
     }
 
+    public static function SCHEDULED(): self
+    {
+        return new self(self::SCHEDULED);
+    }
+
     public function isIdle(): bool
     {
-        return $this->state === self::DONE;
+        return $this->state === self::DONE || $this->state === self::SCHEDULED;
```

Comment:
This is wrong as it means descendants will be executed regardless of if the the scheduled node has run or not - maybe there should be a new state -- RESCHEDULED?

Confusing at least.